Objective-C/Tests/ReplicatorTest+CustomConflict.m

```diff
@@ -233,6 +233,48 @@ - (void) testConflictResolverMergeDoc {
     AssertEqualObjects(savedDoc.toDictionary, exp);
 }
 
+- (void) testDocumentReplicationEventForConflictedDocs {
+    TestConflictResolver* resolver;
+    
+    // when resolution throws exception
```

Comment:
It doesn't throw an exception, it just returns nil...

Reply:
it throws an exception, when we supply a wrong doc id. updated the comment